airbyte-cdk/python/pyproject.toml

```diff
@@ -34,6 +34,8 @@ isodate = "~0.6.1"
 Jinja2 = "~3.1.2"
 jsonref = "~0.2"
 jsonschema = "~3.2.0"
+pandas = "2.0.3"
```

Comment:
Oh, there they are.

We need to figure out if Pandas is okay to tug into the CDK. Could it be that all you need are specific pieces that it uses for xlsx, and they're in a package of their own?

Reply:
I'm not strongly opposed, but we should add pandas to the [file-based extras](https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/pyproject.toml#L79) since it's not required for most connectors